lektor_s3.py

```diff
@@ -142,14 +155,53 @@ def compute_diff(self, local, remote):
                 diff['delete'].append(k)
         return diff
 
+    def create_headers(self, filename):
+        """ Set file headers from mimetype and config. """
+        mime, _ = mimetypes.guess_type(filename)
+        if mime is None:
+            mime = "text/plain"
+        headers = {'ContentType': mime}
+
+        # Set defaults first
+        for upload_arg in TransferManager.ALLOWED_UPLOAD_ARGS:
+            upload_arg_value = self.config.defaults().get(upload_arg.lower())
```

Comment:
This is due to the way [`ConfigParser`](https://docs.python.org/2.7/library/configparser.html) works:

> All option names are passed through the [`optionxform()`](https://docs.python.org/2.7/library/configparser.html#ConfigParser.RawConfigParser.optionxform) method. Its default implementation converts option names to lower case.

But if I can sort it so we use `S3Plugin.get_config` instead of `ConfigParser` then it's a moot point anyway.